metadata.rb

```diff
@@ -6,7 +6,7 @@
 long_description 'Application cookbook for installing and configuring Vault.'
 issues_url 'https://github.com/johnbellone/vault-cookbook/issues'
 source_url 'https://github.com/johnbellone/vault-cookbook/'
-version '2.5.0'
+version '9002.0.0'
```

Comment:
Should the version after "9000" be what the new version would be? ex: `9002.6.0`?

Reply:
It's a major version change, keeping the minor and patch versions would get messy, no?

Reply:
Worth implementing new versioning scheme, or wait until confirmed?

Reply:
Yeah, I'll update with new version.

Reply:
Ok, I've pushed an updated version.